src/libaktualizr/storage/schema_migration_test.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -euo pipefail
+
+if [ $# != 1 ]; then
+    echo "usage: $0 sql_dir" >&2
+    exit 1
+fi
+
+SQL_DIR=$1
+
+DB_CUR=$(mktemp)
+trap 'rm $DB_CUR' exit
+
+DB_MIG=$(mktemp)
+trap 'rm $DB_MIG' exit
+
+if ! [ -f "$SQL_DIR/schema.sql" ]; then
+    echo "schema.sql not found in $SQL_DIR"
+    exit 1
+fi
+
+sqlite3 -batch -init "$SQL_DIR/schema.sql" "$DB_CUR" ";"
+for f in "$SQL_DIR"/migration/migrate.*.sql; do
+    sqlite3 -batch -init "$f" "$DB_MIG" ";"
+done
+
+R=$(sqldiff "$DB_CUR" "$DB_MIG")
```

Comment:
I didn't know that such tool exists. Probably we should replace my horrible hack with `boost::tokenizer` with a shell-out to this tool.

Reply:
Yes, I found it while reading some stackoverflow answer (https://www.sqlite.org/sqldiff.html).
Also good idea, I'll do that.